.github/workflows/consistency-functional.yml

```diff
@@ -4,6 +4,8 @@ on:
     paths-ignore:
       - 'docs/**'
   pull_request:
+  schedule:
+    - cron: '0 */8 * * *'
```

Comment:
Can we add a comment as to when this will run? I always forget cron syntax :(

> At minute 0 past every 8th hour.

Reply:
Yeah this should run every 8 hours

Reply:
done

Reply:
Thanks